managed/ui/src/components/tables/CreateBackup/CreateBackup.js

```diff
@@ -81,21 +81,30 @@ export default class CreateBackup extends Component {
 
     if (isDefinedNotNull(values.storageConfigUUID)) {
       let backupType = null;
+      let frequency = null;
+
       if (this.state.backupType === 'ysql') {
         backupType = YSQL_TABLE_TYPE;
       } else if (this.state.backupType === 'ycql') {
         backupType = YCQL_TABLE_TYPE;
       } else if (this.state.backupType === 'yedis') {
         backupType = YEDIS_TABLE_TYPE;
       }
+
+      if (!isEmptyString(values.schedulingFrequency) && !isEmptyString(values.schedulingFrequencyUnit.value)) {
+        if (values.schedulingFrequencyUnit.value === 'Hours') {
+          frequency = values.schedulingFrequency * 3600 * 1000;
+        } else if (values.schedulingFrequencyUnit.value === 'Minutes') {
+          frequency = values.schedulingFrequency * 60 * 1000;
```

Comment:
Conversion = { HOURS: 3600, MINUTE: 60}

and here.
seelectedValue = values.schedulingFrequencyUnit.value;
frequency = Math.round(values.schedulingFrequency * Conversion[seelectedValue] * 1000);
something like this
so later when we want to add new unite we just need to add in Conversion and contant.

Reply:
Done